### DISCUSSION TITLE
Diff output doesn't use « word diff » like git

### DISCUSSION THREAD

**Discussion by vmaubert** [vitest-dev/vitest#6162] [Feedback]
Hi,

Before the v0.32.1, we had diffs like that: 
![screenshot-0 32 0](https://github.com/user-attachments/assets/c69820b9-84aa-4f80-8eab-62fc1dcccbd2)

After:
![screenshot-0 32 1](https://github.com/user-attachments/assets/e89026c2-4d7e-484a-b187-8d41d69a4f30)

Is it intentional? I find the old version more readable :-)
Same result with macOs or linux.

Thanks!

**Comment by sheremet-va** [comment]
Yes, unfortunately, the previous diff implementation had more bugs, so we switched to the Jest one.

**Comment by vmaubert** [comment]
Hi @sheremet-va, thx for your answer.

I tried with the latest version of Jest, it's like the old version of Vitest =>

![jest](https://github.com/user-attachments/assets/8a1fe690-49ad-42c8-8e68-c5977e629f06)

**Reply by sheremet-va** [reply]
You are right! This should be fixed in https://github.com/vitest-dev/vitest/pull/6191
